src/config/ConfigHelper.ts

```diff
@@ -168,6 +180,17 @@ export const configHelperNetworks: Config[] = [
   }
 ]
 
+export const KNOWN_CONFIDENTIAL_EVMS = {
```

Comment:
why do we need this?
We should always compare or act based on chainId and not on network name

Reply:
`getConfig`
`public getConfig(network: string | number, infuraProjectId?: string): Config {`
...
 uses either network name or chain id... If we supply and search for a "name"  we might not able to check if it is a confidential EVM... specially because we have wrong names on the contracts address.json  ('saphire' misspelled with only 1 'p'). Was trying to overcome this obstacle and make it more flexible, but of course i can put only chain ids on the constant